source/application/IDEAS/IDEASAppController.cpp

```diff
@@ -206,6 +208,9 @@ void IDEASAppController::onEnergyConnected(bool connected){
 
 		connect(xasScanConfiguration, SIGNAL(totalTimeChanged(double)), xasScanConfigurationHolder3_, SLOT(updateOverallScanTime(double)));
 		xasScanConfigurationHolder3_->updateOverallScanTime(xasScanConfiguration->totalTime());
+
+		connect(ideasKETEKDetailedDetectorViewWithSave_, SIGNAL(peakingChanged(const QString&)), xasScanConfigurationView_, SLOT(onDetectorSettingsChange(const QString&)));
+		connect(ideas13ElementGeDetailedDetectorViewWithSave_, SIGNAL(peakingChanged(const QString&)), xasScanConfigurationView_, SLOT(onDetectorSettingsChange(const QString&)));
```

Comment:
Me and @dleacock spent a bit of time on this yesterday. At first glance it looks like wiring a view to another view is a little fragile, but there doesn't seem to be another option with the current  way the detectors are setup in the application. I'm going to keep digging with this, because I feel like there should be a more robust way of listening for this change that by wiring up to the widgets themselves.